### PR TITLE
Add optional argument to skip logging some resources

### PR DESCRIPTION
**Pull request by jlorgal**

### CODE REVIEW

File: lib/logging.js [hunk 1]
```diff
@@ -52,19 +54,41 @@ module.exports = function(logger) {
...
 
+  /**
+   * Check if the request URL starts with any of the blacklist paths.
+   *
+   * @param {String} url
+   *    Request url
+   * @return {Boolean}
+   *    True if the request URL is included in the blacklist.
+   */
+  function isUrlBlackedList(url) {
```
Comment (labajo) [inline lib/logging.js]: isUrlBlackedListed

File: lib/logging.js [hunk 2]
```diff
@@ -52,19 +54,41 @@ module.exports = function(logger) {
...
+    if (blacklist) {
```
Comment (jmendiara) [inline lib/logging.js]: Minor: Consider using a more functional approach to reduce execution branches
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

``` js
// not tested
blacklist = blackist || [];
function isUrlBlackedList(url) {
   return blackist.some(url.startsWith);
}
```

File: lib/logging.js [hunk 3]
```diff
@@ -52,19 +54,41 @@ module.exports = function(logger) {
...
+      }
+    }
+    return false;
+  }
+
   return function loggingMiddleware(req, res, next) {
-    var startTime = new Date();
-    logger.info('Request from %s: %s %s', getClientIp(req), req.method, req.originalUrl);
+    if (!isUrlBlackedList(req.originalUrl)) {
+      var startTime = new Date();
```
Comment (labajo) [inline lib/logging.js]: Date.now()

**Comment by labajo** [comment]
LGTM

**Pull request merged**